### DISCUSSION TITLE
Beta 0.22.0

### DISCUSSION THREAD

**Discussion by AprilSylph** [AprilSylph/XKit-Rewritten#1232] [Tech Office]
- 248912f7 **marcustyphoon** New tweak: Hidden following/mutuals indicators (#1125)
- 328fd26b **marcustyphoon** Implement feature deprecation (#1190)
- 3fc3ebef **marcustyphoon** Interface Util: Prevent processing disconnected elements (#1212)
- eb8b2f8b **marcustyphoon** Show Originals: Fix basic functionality with "virtual scroller" experiment (#1201)
- 92645e62 **marcustyphoon** Tweaks: Prevent hiding pending blaze status label (#1148)
- af2ab3e7 **marcustyphoon** Fix/improve content hiding with "virtual scroller" experiment (#1213)
- b2d7265d **marcustyphoon** Prevent visible UI flicker when processing new posts (#1215)
- 5eeff818 **marcustyphoon** Show Originals: Fix controls element moving on resize (#1220)
- f6ac6f38 **marcustyphoon** Seen Posts: Fix functionality with "virtual scroller" experiment (#1221)
- 518b9db5 **marcustyphoon** Preferences util: Remove local storage loop delay (#1223)
- 2d2a26da **marcustyphoon** Seen Posts: Fix "virtual scroller" experiment (better) (#1225)
- 69750cad **marcustyphoon** Seen Posts: Ignore header/footer in seen post detection (#1006)
- fcf63ccf **marcustyphoon** Quick Tags: Work correctly on non-NPF posts (#969)
- 17615a95 **April** CleanFeed: Add option to treat specified tags as mature (#1185)
- d504ba9d **April Sylph** bump version to 0.22.0

<hr /><em>This discussion was created from the release <a href='https://github.com/AprilSylph/XKit-Rewritten/releases/tag/v0.22.0'>Beta 0.22.0</a>.</em>

**Comment by AprilSylph** [comment]
Ah, that's fun, the release notes get copied into the auto-generated discussion.

I classified this as a minor release rather than a patch release due to the deprecation of Timestamps, meaning the feature list has changed. I don't have the energy to update the wiki right now, but I think we should remove deprecated features from it.

**Comment by AprilSylph** [comment]
Another comment specific to this release... @enchanted-sword, would you consider setting your name on GitHub to something more than just "April"? Even an extra initial would help distinguish between us... and would make the patch notes look less silly :sweat_smile:

I would change the release notes script to print GitHub logins instead, but I don't think those are actually recorded in commits unless the GitHub user doesn't have a name set at all.

**Reply by marcustyphoon** [reply]
Interesting; I may look into this later https://stackoverflow.com/questions/68458886/get-author-login-name-or-username-from-commit

**Reply by enchanted-sword** [reply]
👍
